app/views/layouts/application.html.erb

```diff
@@ -195,6 +195,22 @@
 </div>
 <script type="text/javascript" nonce="<%= content_security_policy_script_nonce %>">
 
+    // error handling for auth errors when getting user access tokens
+    if (window.SCP.userAccessToken === '' && <%= user_signed_in? %>) {
+        var errorTitle = 'Please sign in again';
+        var errorText = '<p class="text-danger">There appears to be an issue with you session - ' +
```

Comment:
you -> your

I like this more general wording.   Maybe make it even shorter 'Your session is no longer available' or 'Your session is no longer valid' ?